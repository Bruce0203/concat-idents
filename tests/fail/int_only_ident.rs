concat_idents::concat_idents!(fn_name = 1 {
    fn fn_name() {
        /* an ident cannot be made of just an integer */
    }
});

fn main() {}
