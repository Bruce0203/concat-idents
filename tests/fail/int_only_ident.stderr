error: Identifies cannot start nor consist only of integers with integers
 --> $DIR/int_only_ident.rs:1:41
  |
1 | concat_idents::concat_idents!(fn_name = 1 {
  |                                         ^
